### ISSUE TITLE
Visual Bugs in MyClaims Page in Mobile version

### ISSUE THREAD

**Issue by drosne** [drosne/RMT#25]
In the mobile version of MyClaims pages there is the same problem that we meet in MyTrips, the flight listing is not matching with the design provided. it's not a big deal i guess but visually it's really crappy currently 
Please notice that for the moment i can't check the rest of that page because in my test page 
'krasi.zlatev@gmail.com / test" I don't see the passengers...

![photo](https://cloud.githubusercontent.com/assets/10349847/5739089/f26b989c-9bf4-11e4-96e5-f8532800c10d.PNG)

Design provided

![rmtck_mob06_my-claims_en-01](https://cloud.githubusercontent.com/assets/10349847/5739096/05155032-9bf5-11e4-96f1-995ce0ddc734.jpg)

**Comment by nppetrov** [comment]
Just got back from the developer.

As for this one, she didn't followed the mobile design in full, since it became unreadable on 320/480. As the html content provided in design is different for "trip details", we can either tweak it a bit to look better as it is, or provide you with separate html file (following mobile design and content) to integrate. Please, let me know which is better for you.

**Comment by drosne** [comment]
I'm not sure to understand what doesn't match with the size ?
But if the design doesn' t match very well .. yes please tweak it to make it work and look the maximum like it is in the initial design

**Comment by nppetrov** [comment]
The mobile AI design has different content than desktop AI design. So we can either tweak what we currently have to fix the original flight plan piece or create separate html with different content which follows the design in whole.

**Comment by drosne** [comment]
I didn't see any difference between designs ?

Romain DROSNE 
Mob : +33 6 62 45 13 64

**Comment by nppetrov** [comment]
Trip details has different content (date, status etc)

**Comment by drosne** [comment]
Yes à little bit but MyTrips and MyClaim have their own design described into the psd templates

Romain DROSNE 
Mob : +33 6 62 45 13 64

**Comment by nppetrov** [comment]
Yes, that's correct. So do you prefer this tweaked in order to fix it with the content from destkop or do you want separate html following exactly following the mobile design (with mobile design content)?

**Comment by drosne** [comment]
tweaked...

Romain 

----------------------------------------------------------------
Romain DROSNE 
Co-Founder

Claim Assistance 
Get up to 800$ for a delayed flight ! 

www.RefundMyTicket.net
www.RembourserMonBillet.com

@RefundMyTicket
----------------------------------------------------------------
